automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -192,6 +192,18 @@ trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually wit
           logger.warn(s"Could not obtain Dataproc log files for cluster ${creatingCluster.googleProject}/${creatingCluster.clusterName}")
       }
 
+      // If the cluster is running, grab the jupyter.log file for debugging.
+      runningOrErroredCluster.foreach { cluster =>
+        if (cluster.status == ClusterStatus.Running) {
+          saveJupyterLogFile(cluster, "create") match {
+            case Success(file) =>
+              logger.info(s"Saved jupyter.log file for cluster ${googleProject.value} / ${clusterName.string} to ${file.getAbsolutePath}")
```

Comment:
It'd be good to start using the method
```
def projectNameString: String = s"${googleProject.value}/${clusterName.value}"
```
in `LeonardoModel` for `automation`, too. Not a blocker, though.